aion/web/views/errors.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Aion.ErrorCodesViews do
```

Comment:
I think it would be easier if the module names resembled the file names, just like they did so far (e.g. RoomChannel -> room_channel.ex, UserRecord -> user_record.ex)

It would be even better if their whole name would represent the filepath, but it's not doable so easily with Phoenix 1.2 architecture. But the file names can mirror the module names at least, it's easier to search through the repo then. 

Also, this is part of a controller, not of a view. I think putting it in the same directory as all the controllers would be alright (the code wouldn't move _that_ far in the whole project directory). You could name it then like `Aion.ControllerErrors` or something similar